src/Bicep.LangServer/Telemetry/BicepTelemetryEvent.cs

```diff
@@ -201,7 +201,50 @@ public static BicepTelemetryEvent CreateDeployStartOrWaitForCompletionResult(str
                 properties: new()
                 {
                     ["deployId"] = deployId,
-                    ["result"] = isSuccess ? Result.Succeeded : Result.Failed
+                    ["result"] = isSuccess ? EventResult.Succeeded : EventResult.Failed
+                }
+            );
+
+        public static BicepTelemetryEvent DecompileSuccess(string decompileId, int countOutputFiles, int countConflictingFiles)
+            => new BicepTelemetryEvent
+            (
+                eventName: "decompile/success",
```

Comment:
Can we please move the eventNames to TelemetryConstants.EventNames?

Reply:
182b2bc3d ("CR fixes", 2022-10-28)